lessons/lesson4.md

```diff
@@ -0,0 +1,145 @@
+## Lesson 4: Ensure Events are published
+
+There are three ways one can consume event
```

Comment:
Which one is recommend and for which case ? Maybe you have use case examples for each of them ?

Reply:
It is more based on user's need, whether they prefer consume by API or runtime action or both, so IMO, there is no recommendation, more based on user story